src/posts/2020-06-08-going-remote.md

```diff
@@ -0,0 +1,110 @@
+
+(Yeah and those Friday references - we allocate our Friday to what we call "investment time". Improving ourselves and our company, learning new languages, frameworks, techniques, trying out weird things, giving talks to each other and so on. More on that later.)
+
+Another powerful tool for battling isolation is to **create actual moments for goofing around**. Every week, at 3 PM, we have a 30 min call to discuss how our previous week was. It's not about work, it's about our personal lives, just to share what we've been doing, how Pedro's house is *still* not finished even though he's been claiming "just this one thing missing" for the past year and a half, how the new Cardi B single is 🔥🔥🔥, you know, the *yoozh*. Every Friday, at the end of the day we have a call to discuss what we accomplished in our investment time. This tends to include people trying out fancy new beers and sharing them. Every day, at the end of the day, there's an opt-in call for folks to just chat. And occasionally you see people on Slack creating calls while they are just coding or doing their *thing*, each on their own project, but all in the same call to mimic the office environment (beware this might hurt your productivity if you need to be focused and should be handled with care). Create and nurture those moments, **it's your team, they're your co-workers** and sometimes we all need to talk about chess and the Sicilian defence being overrated.
+
+So, to reiterate: **don't bank on dailies** - they're a tool for communication within a project, not for isolation. **Care for asynchronous communication**, reply and encourage discussing everyday life on tools like Slack. Share gifs and videos that make you laugh. **Create moments for people to join in and goof around**. And please, if you do have a beer fountain in the office, send a few bottles to your remote coworkers, it's only common courtesy.
+
+### Decision Making
+
+We're a small company that relies on democratic values. We all have the right to an opinion. Company-wide decisions are discussed with everyone and we are free to object. But being remote, For a while I felt **more distant than ever to the decision making process**. This happened because of (1) meetings - discussed ahead, (2) offline conversations, and (3) our transition to Basecamp being rough.
```

Comment:
```suggestion
We're a small company that relies on democratic values. We all have the right to an opinion. Company-wide decisions are discussed with everyone and we are free to object. But being remote, for a while I felt **more distant than ever to the decision making process**. This happened because of (1) meetings - discussed ahead -, (2) offline conversations, and (3) our transition to Basecamp being rough.
```